packages/x-date-pickers/src/internals/hooks/usePicker/usePickerViews.ts

```diff
@@ -28,26 +27,13 @@ export type PickerViewRendererLookup<
 /**
  * Props used to handle the views that are common to all pickers.
  */
-export interface UsePickerViewsBaseProps<TView extends DateOrTimeView> {
+export interface UsePickerViewsBaseProps<TView extends DateOrTimeView>
+  extends Omit<UseViewsOptions<any, TView>, 'onChange' | 'onFocusedViewChange' | 'focusedView'> {
```

Comment:
I did not wanted to add controllable focused view on the picker on this PR.
But that would be very easy to do if we wanted to